cmd/plugin.go

```diff
@@ -314,12 +352,25 @@ func createPluginCommand() *cobra.Command {
 		log.Error().Str("module", "plugin").Err(err)
 	}
 
+	enableCommand.PersistentFlags().StringP("plugin", "p", "", "Installed plugin name")
+	err = viper.BindPFlag("enable_plugin", enableCommand.PersistentFlags().Lookup("plugin"))
+	if err != nil {
+		log.Error().Str("module", "plugin").Err(err)
+	}
+
+	disableCommand.PersistentFlags().StringP("plugin", "p", "", "Installed plugin name")
+	err = viper.BindPFlag("disable_plugin", disableCommand.PersistentFlags().Lookup("plugin"))
+	if err != nil {
+		log.Error().Str("module", "plugin").Err(err)
+	}
+
 	cmd.AddCommand(statusCommand)
 	cmd.AddCommand(installCommand)
 	cmd.AddCommand(uninstallCommand)
 	cmd.AddCommand(startCommand)
 	cmd.AddCommand(stopCommand)
 	cmd.AddCommand(enableCommand)
+	cmd.AddCommand(disableCommand)
```

Comment:
This is good to add commands more.  
I've refactored the command addition section using a loop. By storing all command objects in a slice and iterating over them, we can add each command to the root command with a single loop. This approach not only cleans up the code, making it more readable, but also simplifies any future modifications or additions to the command list.
This change might be a matter of personal preference, so it's ultimately up to your decision.

```
commands := []*cobra.Command{
    statusCommand,
    installCommand,
    uninstallCommand,
    startCommand,
    stopCommand,
    enableCommand,
    disableCommand,
    listCommand,
}

for _, cmd := range commands {
    rootCmd.AddCommand(cmd)
}
```